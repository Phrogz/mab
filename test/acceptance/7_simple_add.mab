p( "Test#7a" )
setSlot( "add", method( a, b,
	a +( b )
) )
p( add(40,2) )