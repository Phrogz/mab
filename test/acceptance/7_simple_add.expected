Test#7a
42